libs/sdk-core/src/lib/db/resource/resource.ts

```diff
@@ -31,10 +31,20 @@ import { setEntities, setLabels, sliceUnicode } from './resource.helpers';
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface ReadableResource extends IResource {}
+
+/** Implements all the read operations on resources.
+ *
+ * A resource allows you to store content in the Knowledge Box.
+ * A single resource might contain several fields.
+ *
+ * Fields have different types: files, links, texts, conversations, etc.
+ */
 export class ReadableResource implements IResource {
   data: ResourceData = {};
   private fieldTextsCache: { [key: string]: string[] } = {};
 
+  /** Note: Usually you will not need to create a `Resource` object yourself.
+   * It is returned by the `getResource` method of the `KnowledgeBox` object. */
```

Comment:
is it possible to make this `getResource` navigable to the right documentation page?